### PR TITLE
Use Alpine as root image

### PR DESCRIPTION
**Pull request by harmw**
This supports #794 , though right now it only works for the main backend/frontend containers. I haven't looked at the supporting stuff, like _something mssql_ I noticed :)

### CODE REVIEW

**Comment by Hironsan** [comment]
Thank you @harmw . I was suprised that the images are small(backend: 1.22GB -> 215MB, frontend: 1.25GB -> 479MB)

By the way, I executed the following docker-compose command. The build was successful but the running was failed:

```
> docker-compose -f docker-compose.prod.yml up
...
Successfully built 2f567b196783
Successfully tagged doccano_nginx:latest
WARNING: Image for service nginx was built because it did not already exist. To rebuild this image you must use `docker-compose build` or `docker-compose up --build`.
Creating doccano_frontend_1 ... done
Creating doccano_postgres_1 ... done
Creating doccano_backend_1  ... done
Creating doccano_nginx_1    ... done
Attaching to doccano_postgres_1, doccano_frontend_1, doccano_backend_1, doccano_nginx_1
backend_1   | env: can't execute 'bash': No such file or directory
postgres_1  | 2020-06-09 06:25:30.206 UTC [1] LOG:  starting PostgreSQL 12.0 on x86_64-pc-linux-musl, compiled by gcc (Alpine 8.3.0) 8.3.0, 64-bit
postgres_1  | 2020-06-09 06:25:30.207 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres_1  | 2020-06-09 06:25:30.207 UTC [1] LOG:  listening on IPv6 address "::", port 5432
nginx_1     | 2020/06/09 06:25:31 [emerg] 1#1: host not found in upstream "backend" in /etc/nginx/conf.d/nginx.conf:15
nginx_1     | nginx: [emerg] host not found in upstream "backend" in /etc/nginx/conf.d/nginx.conf:15
postgres_1  | 2020-06-09 06:25:30.210 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres_1  | 2020-06-09 06:25:30.225 UTC [18] LOG:  database system was shut down at 2020-06-09 05:59:59 UTC
postgres_1  | 2020-06-09 06:25:30.229 UTC [1] LOG:  database system is ready to accept connections
doccano_frontend_1 exited with code 0
doccano_backend_1 exited with code 127
doccano_nginx_1 exited with code 1
```

There may be a problem in backend. As a result, nginx container exited with code 1.

**Comment by harmw** [comment]
```
backend_1   | env: can't execute 'bash': No such file or directory
```
hm, did I miss that somewhere? Things on my  end _just  worked_, given not using `docker-compose`. Let me take a look, I did notice an unrelated issue with the `frontend` failing to start when using the `prod` compose, but didn't bother looking into that as I had that problem also in `master`.

**Comment by Hironsan** [comment]
When I changed Dockerfile and run.sh, it works fine.

Dockerfile(install bash)
```
RUN apk add -U python3 python3-dev libpq postgresql-dev unixodbc-dev musl-dev g++ libffi-dev bash\
```

run.sh
- python -> python3

But I don't want to change run.sh as much as possible.

**Comment by harmw** [comment]
```
$ docker-compose -f docker-compose.prod.yml up
Creating network "doccano_default" with the default driver
Creating network "doccano_network-backend" with the default driver
Creating network "doccano_network-frontend" with the default driver
Creating doccano_postgres_1 ... done
Creating doccano_frontend_1 ... done
Creating doccano_backend_1  ... done
Creating doccano_nginx_1    ... done
Attaching to doccano_postgres_1, doccano_frontend_1, doccano_backend_1, doccano_nginx_1
backend_1   | Making staticfiles
postgres_1  | 2020-06-10 10:37:45.504 UTC [1] LOG:  starting PostgreSQL 12.0 on x86_64-pc-linux-musl, compiled by gcc (Alpine 8.3.0) 8.3.0, 64-bit
postgres_1  | 2020-06-10 10:37:45.505 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres_1  | 2020-06-10 10:37:45.506 UTC [1] LOG:  listening on IPv6 address "::", port 5432
postgres_1  | 2020-06-10 10:37:45.511 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres_1  | 2020-06-10 10:37:45.542 UTC [18] LOG:  database system was interrupted; last known up at 2020-05-23 14:10:53 UTC
postgres_1  | 2020-06-10 10:37:45.682 UTC [18] LOG:  database system was not properly shut down; automatic recovery in progress
postgres_1  | 2020-06-10 10:37:45.685 UTC [18] LOG:  redo starts at 0/178E738
postgres_1  | 2020-06-10 10:37:45.685 UTC [18] LOG:  invalid record length at 0/178E770: wanted 24, got 0
postgres_1  | 2020-06-10 10:37:45.685 UTC [18] LOG:  redo done at 0/178E738
postgres_1  | 2020-06-10 10:37:45.695 UTC [1] LOG:  database system is ready to accept connections
doccano_frontend_1 exited with code 0
backend_1   |
backend_1   | 0 static files copied to '/app/staticfiles', 315 unmodified, 1009 post-processed.
backend_1   | Initializing database
backend_1   | Operations to perform:
backend_1   |   Apply all migrations: admin, api, auth, authtoken, contenttypes, sessions, social_django
backend_1   | Running migrations:
backend_1   |   No migrations to apply.
backend_1   |   Your models have changes that are not yet reflected in a migration, and so won't be applied.
backend_1   |   Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
backend_1   | CommandError: Error: That username is already taken.
backend_1   | Starting django
backend_1   | [2020-06-10 10:38:27 +0000] [26] [INFO] Starting gunicorn 19.9.0
backend_1   | [2020-06-10 10:38:27 +0000] [26] [INFO] Listening at: http://0.0.0.0:8000 (26)
backend_1   | [2020-06-10 10:38:27 +0000] [26] [INFO] Using worker: sync
backend_1   | [2020-06-10 10:38:27 +0000] [29] [INFO] Booting worker with pid: 29
```

That's not what you got, let me check if maybe I got some outdated containers dangling around 🤔 

Let me try again and start with a wipe of my docker-box.

**Comment by harmw** [comment]
should be fine now 👍
